locale/en/blog/vulnerability/openssl-november-2022.md

```diff
@@ -23,8 +23,8 @@ the _highest_ severity issue is CRITICAL.
 
 [security policy](https://www.openssl.org/policies/secpolicy.html).
 
-Node.js v18.x and v19.x use OpenSSL v3.
-Therefore these release lines are impacted by this update.
+Node.js v17.x, v18.x and v19.x use OpenSSL v3.
+While Node.js v17.x is no longer supported, the v18.x and v19.x release lines are impacted by this update.
```

Comment:
```suggestion
Node.js v17.x, v18.x, and v19.x use OpenSSL v3.

Node.js v18.x and v19.x will be updated to address this issue.

Support for Node.js v17.x ended in June 2022. It will not be updated. Please migrate to a supported version of Node.js.
```